src/main/java/com/ldtteam/structurize/storage/rendering/types/BlueprintPreviewData.java

```diff
@@ -108,15 +109,23 @@ public BlueprintPreviewData(final FriendlyByteBuf byteBuf, final boolean serverS
         pos = byteBuf.readBlockPos();
         this.packName = byteBuf.readUtf(32767);
         this.blueprintPath = byteBuf.readUtf(32767);
+        rotationMirror = RotationMirror.values()[byteBuf.readByte()];
+    }
+
+    /**
+     * Prepares blueprint if possible
+     */
+    public BlueprintPreviewData prepareBlueprint(final HolderLookup.Provider provider)
```

Comment:
Where is this called then?

Reply:
this is probably unnecessary, might revert
but its called from messages just like it was, was just context issue